src/core/dom/media-query-props.js

```diff
@@ -98,7 +98,7 @@ export class MediaQueryProps {
    * @return {string} value
    */
   resolve_(exprString, parser, emptyExprValue) {
-    if (!exprString || !exprString.trim()) {
+    if (!exprString?.trim()) {
```

Comment:
The types say this is always a string, so `?.` is unnecessary.

Reply:
Removed